src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/lib/__tests__/pipeline.js

```diff
@@ -0,0 +1,200 @@
+import _ from 'lodash';
```

Comment:
This only tests remove, add, moveup and movedown right now. I'd like to see tests for the other methods as well, especially `applySimulateResults`